packages/perseus-editor/src/components/sortable.tsx

```diff
@@ -1,10 +1,24 @@
 import {css, StyleSheet} from "aphrodite";
-import createReactClass from "create-react-class";
-import PropTypes from "prop-types";
 import * as React from "react";
 import ReactDOM from "react-dom";
 
-const PT = PropTypes;
+// Need to update these values.
```

Comment:
Could this be a TODO comment? Or are you planning to resolve it before this PR lands?

```suggestion
// TODO(catjohnson): update these values.
```